app/viewmodels/CYASummaryListHelper.scala

```diff
@@ -445,4 +445,40 @@ class CYASummaryListHelper {
     } else None
   }
 
+  def buildDefermentSummaryList(answer: UserAnswers)(implicit messages: Messages): Option[CYASummaryList] = {
+    val paymentInformationSummaryListRow: Option[Seq[SummaryListRow]] = answer.get(DefermentPage) map { deferment =>
+      val payingByDeferment = if (deferment) "Yes" else "No"
```

Comment:
Could the "yes" and "no" be messages("site.yes"), and message("site.no")?

This is just a recommendation.. In two minds with which one is better to use for this